Aztec/Classes/Public/TextKit/TextAttachment.swift

```diff
@@ -116,6 +139,39 @@ public class TextAttachment: NSTextAttachment
 
         return CGRect(origin: CGPointZero, size: CGSize(width: width, height: onScreenHeight(width)))
     }
+
+    func startImageFetching() {
+        var remoteURL: NSURL?
```

Comment:
You can change this to `let remoteURL: NSURL` and it'll work too.

Also consider renaming the variable since the URL may not be "remote" in some cases.

The advantage is we won't have an optional here.